sernet.gs.service/src/sernet/verinice/service/commands/CopyCommand.java

```diff
@@ -276,6 +276,8 @@ private CnATreeElement saveCopy(CnATreeElement toGroup, CnATreeElement elementTo
         CnATreeElement newElement = saveNew(toGroup, copyElement);
         if(newElement.getEntity()!=null) {
```

Comment:
Make sure that the old copy operation remains unchanged if the following applies: `module-templates.active=false`

Please remove the next if statement from this command and implement another solution.